src/Model/Category/Category.php

```diff
@@ -148,97 +112,49 @@ public function setParentId($parentId)
     /**
      * @return string
      */
-    public function getName()
+    public function getTitle()
     {
-        return $this->name;
+        return $this->title;
     }
 
     /**
-     * @param string $name
+     * @param string $title
      */
-    public function setName($name)
+    public function setTitle($title)
     {
-        $this->name = $name;
+        $this->title = $title;
     }
 
     /**
      * @return string
      */
-    public function getUrl()
+    public function getPath()
     {
-        return $this->url;
+        return $this->path;
     }
 
     /**
-     * @param string $url
+     * @param string $path
      */
-    public function setUrl($url)
+    public function setPath($path)
     {
-        $this->url = $url;
+        $this->path = $path;
     }
 
     /**
      * @return string
      */
-    public function getImageUrl()
-    {
-        return $this->imageUrl;
-    }
-
-    /**
-     * @param string $imageUrl
-     */
-    public function setImageUrl($imageUrl)
-    {
-        $this->imageUrl = $imageUrl;
-    }
-
-    /**
-     * @return string
-     */
-    public function getThumbnailImageUrl()
-    {
-        return $this->thumbnailImageUrl;
-    }
-
-    /**
-     * @param string $thumbnailImageUrl
-     */
-    public function setThumbnailImageUrl($thumbnailImageUrl)
-    {
-        $this->thumbnailImageUrl = $thumbnailImageUrl;
-    }
-
-    /**
-     * @return bool
-     */
-    public function getVisibleInMenu()
-    {
-        return $this->visibleInMenu;
-    }
-
-    /**
-     * @param $visibleInMenu
-     */
-    public function setVisibleInMenu($visibleInMenu)
-    {
-        $this->visibleInMenu = $visibleInMenu ? true : false;
-    }
-
-    /**
-     * @return string
-     */
-    public function getLevel()
```

Comment:
So we no longer need those?
It looks like you're missing that the category listing is stored separately from category information, so shouldn't this be a separate entity for CMP data?

Reply:
I have asked previously wether category tagging has been used. From what I understood, it is not. Even though category tagging is implemented (checked on M2 and Presta) it was never really used anywhere. And now, we do not need these fields according to spec.